ts/mains/main_node.ts

```diff
@@ -1023,6 +1023,60 @@ ipc.on('get-start-in-tray', event => {
   }
 });
 
+ipc.on('set-auto-start-enabled', (event, newValue) => {
+  try {
+    userConfig.set('autoStart', newValue);
+
+    // Set the login item settings based on the platform
+    if (process.platform === 'darwin') {
+      // macOS
+      app.setLoginItemSettings({
+        openAtLogin: newValue,
+        openAsHidden: false,
+      });
+    } else if (process.platform === 'win32') {
+      // Windows - For Squirrel-based apps, we need to handle the stub launcher - https://www.electronjs.org/docs/latest/api/app/#appsetloginitemsettingssettings-macos-windows
+      const appFolder = path.dirname(process.execPath);
+      const ourExeName = path.basename(process.execPath);
+      const stubLauncher = path.resolve(appFolder, '..', ourExeName);
+
+      app.setLoginItemSettings({
+        openAtLogin: newValue,
+        path: stubLauncher,
+        args: [],
+      });
+    } else {
+      // Linux and other platforms - basic support
+      app.setLoginItemSettings({
+        openAtLogin: newValue,
+      });
```

Comment:
we don't support that right? Maybe throw?

Reply:
yeah sorry this was meant to be gone, i included it to test it, can confirm it doesnt do anything on ubuntu